components/Footer.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import Container from 'react-bootstrap/Container';
+
+// primary background
+// links color secondary
+
+const Footer = () => {
+  return (
+    <footer>
+      <Container>
+        <div className='footer-wrapper'>
```

Comment:
footer--wrapper class instead of footer-wrapper

Reply:
Changed